tests/dummy/app/components/data-test-component.js

```diff
@@ -0,0 +1,5 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  attributeBindings: [],
```

Comment:
@bgentry good point, thanks. I've added a code comment explaining why we override this property.